src/tests/end-to-end/tests/details-view/fast-pass-report.test.ts

```diff
@@ -0,0 +1,103 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { FeatureFlags } from 'common/feature-flags';
+import { Download } from 'playwright';
+import {
+    detailsViewSelectors,
+    fastPassReportSelectors,
+    tabStopsSelectors,
+} from 'tests/end-to-end/common/element-identifiers/details-view-selectors';
+import { BackgroundPage } from 'tests/end-to-end/common/page-controllers/background-page';
+import { HtmlReportPage } from 'tests/end-to-end/common/page-controllers/html-report-page';
+import { getTestResourceUrl } from 'tests/end-to-end/common/test-resources';
+import { Browser } from '../../common/browser';
+import { launchBrowser } from '../../common/browser-factory';
+import { DetailsViewPage } from '../../common/page-controllers/details-view-page';
+import { TargetPage } from '../../common/page-controllers/target-page';
+import { scanForAccessibilityIssues } from '../../common/scan-for-accessibility-issues';
+
+const reportDownloadUrl = getTestResourceUrl('fast-pass-report.html');
+
+describe('Details View -> FastPass -> Report', () => {
+    let browser: Browser;
+    let targetPage: TargetPage;
+    let detailsViewPage: DetailsViewPage;
+    let backgroundPage: BackgroundPage;
+    let reportPage: HtmlReportPage;
+    const reportSaveAsFilePath = 'src/tests/end-to-end/test-resources/fast-pass-report.html';
```

Comment:
I think it would be preferable to avoid having a test mutate a file under `src` at runtime (unless it's through an explicit snapshotting mechanism). Could we instead just omit the `saveAs` call entirely and use `await reportDownload.path()` as the `reportDownloadUrl`?

Reply:
Per our conversation, I moved it to use the temp `test-results` directory.